homeassistant/components/glances/config_flow.py

```diff
@@ -27,7 +27,6 @@
     DEFAULT_SCAN_INTERVAL,
     DEFAULT_VERSION,
     DOMAIN,
-    SUPPORTED_VERSIONS,
```

Comment:
In my opinion this would be better as a separate PR

Reply:
Sure I can have in a separate PR. I just didn't want to increase the pile of PRs

Reply:
PR #77541

Reply:
Better to pile small PRs than to have a big ondes stick in queue for ages